src/main/scala/beam/agentsim/agents/modalbehaviors/ChoosesMode.scala

```diff
@@ -24,14 +24,14 @@ import beam.sim.{BeamServices, Geofence}
 import beam.sim.population.AttributesOfIndividual
 import beam.utils.plan.sampling.AvailableModeUtils._
 import com.vividsolutions.jts.geom.Envelope
-import org.matsim.api.core.v01.population.{Activity, Leg}
+import org.matsim.api.core.v01.population.{Activity, Leg, Person}
```

Comment:
you don't need to add this `Person` import since you are `toString`'ing the Id below